app/commands/unlock_command.rb

```diff
@@ -9,8 +9,26 @@ def run
       return Slash.reply(ValidationErrorMessage.build(record: env)) if env.invalid?
 
       slashdeploy.unlock_environment(user.user, env)
-      Slash.say UnlockedMessage.build \
-        environment: env
+      new_active_lock = slashdeploy.give_lock_to_next_user(env)
+
+      if new_active_lock
+        Thread.new do
```

Comment:
Also, I think we should put this within `SlashDeploy::Service#give_lock_to_next_user` so if it gets used elsewhere (e.g. from controllers in a web ui), the slack message still gets sent properly.